### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by AlirezaAkbarix** [gk5885/dagger-android-sample#7]
I think it's time to update the sample
You're using these dependencies:
 `classpath 'com.android.tools.build:gradle:0.14.1' //!!!!!!
  compile 'com.google.dagger:dagger:2.0-SNAPSHOT'
  apt 'com.google.dagger:dagger-compiler:2.0-SNAPSHOT'
  provided 'org.glassfish:javax.annotation:10.0-b28'`

As your sample is highly referenced, I suggest update this repo. thanks.

**Comment by gk5885** [comment]
You're correct that this sample is woefully out of date.  It is also incomplete.  The Dagger team is working on a full example that we plan to finish by the end of the year.

**Comment by ItsHarper** [comment]
Was that sample ever finished?

**Comment by sophiataskova** [comment]
Checking in again! I've hard the hardest time trying to find any google-approved example usage of dagger in Android. For example, this sample seems to rely on module instances, which supposedly aren't recommended and the team has "considered removing" https://google.github.io/dagger/semantics/
